tests/functional/adapter/test_ha_iceberg.py

```diff
@@ -62,7 +62,7 @@ def test__table_creation(self, project, capsys):
         out, _ = capsys.readouterr()
         # in case of 2nd run we expect that the target table is renamed to __bkp
         alter_statement = (
-            f"alter table `awsdatacatalog`.`{project.test_schema}`.`{relation_name}` "
+            f"alter table `{project.test_schema}`.`{relation_name}` "
```

Comment:
Technically this works. Iceberg DDL supports the catalog component in relation names. I couldn't figure out if there was a way to detect whether a table was Iceberg or Hive in `AthenaRelation`. If there is, we can make the names include the catalog for Iceberg tables.

At the same time, I think it's unnecessary. The catalog is defined in the catalog (database) is a required configuration parameter (I think) and is passed in the `QueryExecutionContext.Catalog` property for every Athena query. That makes it redundant to include in table names.

Reply:
I'm happy to leave it as you suggested, I find the `awsdatacatalog` redundant. Specifically for the reason that you mentioned, and because the "writing" capabilities are only in the "awsdatacatalog".